lib/zendesk_apps_support/validations/secure_settings.rb

```diff
@@ -15,6 +15,13 @@ def call(package)
 
           secure_or_hidden_default_param_found = manifest_params.any? { |param| secure_or_hidden_default_param?(param) }
           package.warnings << hidden_default_parameter_warning if secure_or_hidden_default_param_found
+
+          secure_params_without_scopes = manifest_params.filter_map do |param|
```

Comment:
I just realized that this warning only makes sense when the `apps_secure_setting_scopes` arturo is on. This should be provided by the `validate_scopes_for_secure_parameter` in https://github.com/zendesk/zendesk_apps_support/blob/6d6aa8c475a51e1c75568020d7a6939d200b3738/lib/zendesk_apps_support/validations/manifest.rb#L17

I updated the story in Jira with the arturo information.